### DISCUSSION TITLE
How to use Panning on SvgCanvas ?

### DISCUSSION THREAD

**Discussion by DAOLIDEWONIU** [SVG-Edit/svgedit#541] [Q&A]
**Describe the bug**
Hello, how do I use Panning on SvgCanvas ？

**Desktop (please complete the following information):**
 - OS: Tried on Solus Budgie and Windows 10
 - Browser  Chrome (72 and 73), firefox (64)

**Comment by jfhenon** [comment]
Question like this one should now be requested on stackoverflow https://stackoverflow.com/questions/tagged/svg-edit
or the discussion board https://github.com/SVG-Edit/svgedit/discussions.
Thanks
